### ISSUE TITLE
Is that fair to compare with BLIP2 which freeze the image encoder in the Stage 1 and Stage 2

### ISSUE THREAD

**Issue by viyjy** [yiren-jian/BLIText#1]
Hi, the idea of this paper is quite interesting, thanks.
One concern is that BLIP2 freezes the image encoder in the Stage 1 and Stage 2, but your model doesn't. Given this, is the comparison fair?

**Comment by yiren-jian** [comment]
Thank you for your interest!

We freeze ViT as well (surly), see [here](https://github.com/yiren-jian/BLIText/blob/main/lavis/models/blip2_models/blip2_pformer_opt.py#L64). Also, if ViT is not frozen, we can not train with such large batch size without gradient checkpointing and unfreezing ViT will make the pre-training (impractically) very long.

I need to apologize we did not include the snowflake sign for ViT in our Figure 1 and 2, which probably leads to your confusion?

**Comment by viyjy** [comment]
Thanks. Yes, Figure 1&2 triggers this question.
